src/main/java/com/extendaretail/vertx/gcp/pubsub/v1/PubSubMessage.java

```diff
@@ -2,11 +2,14 @@
 
 import io.vertx.codegen.annotations.DataObject;
 import io.vertx.codegen.annotations.Fluent;
+import io.vertx.core.buffer.Buffer;
 import io.vertx.core.json.JsonObject;
 import java.util.Map;
 
 /**
- * Value object to send a PubSub message via {@link PubSubService}
+ * Value object to send a PubSub message via {@link PubSubService}. While you can set both a Vertx
+ * {@link JsonObject} and {@link Buffer} for the payload type, the {@link Buffer} takes precedence
```

Comment:
Place this javadoc (relating to precedence) in the service implementation instead

Reply:
Done 👍